blaze/compute/sql.py

```diff
@@ -620,11 +644,24 @@ def compute_up(t, s, **kwargs):
     return select(s).limit(t.n)
 
 
-@dispatch(Label, ClauseElement)
+@dispatch(Label, ColumnElement)
 def compute_up(t, s, **kwargs):
     return s.label(t.label)
 
 
+@dispatch(Label, FromClause)
+def compute_up(t, s, **kwargs):
+    assert len(s.c) == 1, \
```

Comment:
No, this is me expressing an assumption about the code that follows it. If it fails, then I've hit a case that I didn't handle. The common case is that I've (re)labeled the output of a column that's from a more complex statement than just selecting the column